src/llv8.cc

```diff
@@ -955,6 +961,15 @@ std::string HeapNumber::Inspect(Error& err) {
 }
 
 
+std::string Symbol::ToString(Error& err) {
+  if (!String::IsString(v8(), Name(err), err)) {
```

Comment:
Any reason not to do `Symbol()`/`Symbol('foo')`, like would be shown in JS?

Reply:
You mean for unnamed symbols? `Symbol()` sounds like a good idea in this case.